src/shared/tests.h

```diff
@@ -200,3 +200,35 @@ static inline int run_test_table(void) {
         DEFINE_TEST_MAIN_FULL(log_level, intro, NULL)
 #define DEFINE_TEST_MAIN(log_level)                     \
         DEFINE_TEST_MAIN_FULL(log_level, NULL, NULL)
+
+#define ASSERT_OK(expr)                                                                  \
+    do {                                                                                 \
+        if (!(expr)) {                                                                   \
+            sd_journal_print(LOG_ERR, "Assertion failed: %s", #expr);                    \
+            abort();                                                                     \
+        }                                                                                \
+    } while (false)
+
+#define ASSERT_EQ(actual, expected)                                                      \
+    do {                                                                                 \
+        if ((actual) != (expected)) {                                                    \
+            sd_journal_print(LOG_ERR, "Assertion failed: %s != %s", #actual, #expected); \
```

Comment:
IMHO this is useless. If I looked at such a failure message, I'd want to se the _values_ the expressions evaluated into, not the expressions themselves (I can see those in the code).

Reply:
Indeed, please make sure the actual values are shown. Also, do not use `sd_journal_print`, use `log_error`

Reply:
Thanks for the review.